distribution/server/src/assemble/LICENSE.bin.txt

```diff
@@ -360,10 +360,9 @@ The Apache Software License, Version 2.0
     - io.netty-netty-resolver-4.1.43.Final.jar
     - io.netty-netty-resolver-dns-4.1.43.Final.jar
     - io.netty-netty-transport-4.1.43.Final.jar
-    - io.netty-netty-transport-native-epoll-4.1.43.Final-linux-x86_64.jar
+    - io.netty-netty-transport-native-epoll-4.1.33.Final-linux-x86_64.jar
```

Comment:
Not directly related with this PR but this dependency doesn't look it's getting picked up correctly, we should pin the version correctly to avoid incompatibilities across versions.

Reply:
yes will do it in a separate pull request.